integration-tests/src/test/java/org/apache/fineract/integrationtests/SchedulerJobsTestResults.java

```diff
@@ -263,8 +263,7 @@ public void testTransferFeeForLoansFromSavingsJobOutcome() throws InterruptedExc
         final Integer loanProductID = createLoanProduct(null);
         Assertions.assertNotNull(loanProductID);
 
-        final Integer loanID = applyForLoanApplication(clientID.toString(), loanProductID.toString(), savingsId.toString(),
-                "10 January 2013");
```

Comment:
This date changes were done because the test was invalid (expected disbursement date was earlier than approval date) but there was no proper validation prior, right?

Reply:
yes, the previous code missed these validations, but since those have been improved, these tests with the incorrect dates started to fail..